### ISSUE TITLE
Set the position of app title

### ISSUE THREAD

**Issue by trihandhoko** [blackshadowsoftwareltd/animation_search_bar#2]
Please adding feature to set the app title is on center or left side, because not all page the title in the center. thank you :)

**Comment by philitell** [comment]
Yes please, this would be great. Maybe it would be even better to be able to assign a Widget to centerTitle and not only a String. This would give more room for customizations.
